extensions/amp-video/0.1/amp-video.js

```diff
@@ -774,6 +777,7 @@ export class AmpVideo extends AMP.BaseElement {
     if (!captionsElement) {
       return;
     }
+    this.captionsAreDelegated_ = true;
```

Comment:
`delegated` by itself is confusing here, since the HTMLVideoElement is still responsible for loading, parsing, playing the captions. Only the rendering is custom.